src/patterns/__snapshots__/api-lambda.test.ts.snap

```diff
@@ -523,7 +523,7 @@ Object {
         "lambdaapiANY4EBCADD1",
       ],
       "Properties": Object {
-        "Description": "Automatically created by the RestApi construct",
+        "Description": "this is a test",
```

Comment:
This change is a welcomed fix from [v2.34.0](https://github.com/aws/aws-cdk/releases/tag/v2.34.0), specifically https://github.com/aws/aws-cdk/pull/21207.